packages/boilerplate-generator/package.js

```diff
@@ -3,6 +3,10 @@ Package.describe({
   version: '1.3.1'
 });
 
+Npm.depends({
+  "combine-streams": "1.0.0"
```

Comment:
I worry that the `combine-streams` package is not very heavily developed/starred/downloaded. In particular, it doesn't seem to pay any attention to the string encoding of the written chunks.

After a bit of digging, this [`combined-stream2`](https://www.npmjs.com/package/combined-stream2) package looks a bit more mature.